src/components/file-dropzone/file-dropzone.scss

```diff
@@ -1,5 +1,5 @@
 :host(limel-file-dropzone) {
-    display: grid;
+    display: block;
```

Comment:
What if we use display: flex here?

Reply:
A "block element" is an HTML element that typically starts on a new line and takes up the full width available to it. This way, it creates a block-like structure on the web page. 

Examples of block-level elements include `<div>`, `<p>`, `<header>`, `<footer>`, `<section>`, and `<article>`. These elements are often used for structuring and organizing content on a webpage. 

Since this component is a "wrapper", it shouldn't automatically affect the layout of its children. It should be neutral. The only neutral thing we have is `block`, which will turn it into a block element. It's essentially telling the web-browser that it should treat `limel-file-dropzone` like a `div`.

Both `gird` and `flex` affect how their children are distributed in the layout. Grid forces the children to be rendered in rows (vertically), and Flex forces its children to be rendered in columns (horizontally). If we use this, we will have to document this behavior, and ask the consumers to override this or think about it every time they use this component.

Sometimes, the consumers should still override the layout, buy setting `display: something-else;` on `<limel-file-dropzone />`. This is what we did in the web-client, since there was 3 children there already and they were set-up to be flex children. So we turned `limel-file-dropzone` to a `flex` there.

Reply:
Thanks, Kia! I was not sure since we are overriding it in the webclient. However, your explanations sound reasonable, let me test it on both sides one more time. Yesterday it broke the webclient layout.

Reply:
thanks Lucy. Just make sure you're on the latest on the web client and crm components. That problem is already fixed and merged there, and is independent of this change here.